package.json

```diff
@@ -41,6 +41,7 @@
     "babel-plugin-transform-es2015-modules-amd": "^6.24.0",
     "bower-config": "^1.3.0",
     "bower-endpoint-parser": "0.2.2",
+    "broccoli-assembler": "twokul/broccoli-assembler",
```

Comment:
Will need a release before we can merge here

Reply:
absolutely! Maybe it would make sense to move it under ember-cli umbrella? Let me know.